go.mod

```diff
@@ -16,3 +16,5 @@ require (
 	google.golang.org/genproto v0.0.0-20201109203340-2640f1f9cdfb // indirect
 	google.golang.org/grpc v1.33.2 // indirect
 )
+
+replace github.com/heimweh/go-pagerduty => github.com/mrdubr/go-pagerduty v0.0.0-20221214154643-5a5f6734ec0d
```

Comment:
Hey @mrdubr [Dependant PR got merged](https://github.com/heimweh/go-pagerduty/pull/106), so now You can drop this Replace and update to the corresponding version of **go-pagerduty** ✌🏽